### PR TITLE
Merge plugin consolidation

### PR DESCRIPTION
**Pull request by stratease**
### 🗒️ Description
Merge OM into bucket.

### ✔️ Checklist
- [ ] Changelog entry in the `readme.txt` file.
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: tests/_bootstrap.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	 */
 
 	// TEC Test Support.
-	$tec_dir         = dirname( __DIR__, 2 );
+	$tec_dir         = dirname( __DIR__, 3 ) . '/the-events-calendar';
```
Comment (Camwyn) [inline tests/_bootstrap.php:19]: You shouldn't assume we are in the TEC directory!

**Review by Camwyn — APPROVED** [review]

**Comment by Camwyn** [comment]
I've skipped all the EVA tests as I don't think they belong here - they were failing due to missing classes from parent plugins and such.

File: tests/_bootstrap.php (hunk 1 above)
Comment (stratease) [reply tests/_bootstrap.php:19]: @Camwyn  It is some workaround for EVA tests. Not sure we should mess with this at this point. cc @pattihis

Comment (Camwyn) [reply tests/_bootstrap.php:19]: The test is skipped for now, so the heck with it. It should work the same.

**Review by Camwyn — APPROVED** [review]

**Pull request merged**